apps/estuary/runtime.exs

```diff
@@ -40,3 +40,7 @@ config :estuary,
 
 config :logger,
   level: :warn
+
+config :yeet,
+  endpoint: endpoints,
+  topic: System.get_env("DLQ_TOPIC")
```

Comment:
We will need to add this to the chart and deploy repo as a new environment variable that gets passed in.

Reply:
agreed